crates/symbolicator-service/src/services/download/mod.rs

```diff
@@ -322,13 +322,14 @@ impl DownloadService {
             Ok(Ok(res)) => res,
         };
 
+        metric!(counter("service.download.failure") += 1, "source" => &source_metric_key);
```

Comment:
this should still match on `result` first, otherwise you emit this for successes as well :-)

Another idea would also be to explicitly emit errors for errors that happen on the internal project source.

Reply:
Derp